src/current/_includes/molt/migration-prepare-database.md

```diff
@@ -125,31 +130,160 @@ GRANT SELECT, FLASHBACK ON migration_schema.tbl TO MIGRATION_USER;
+{{site.data.alerts.callout_info}}
+Connect to the primary PostgreSQL instance, **not** a read replica. Read replicas cannot create or manage logical replication slots. Verify that you are connected to the primary server by running `SELECT pg_is_in_recovery();` and getting a `false` result.
+{{site.data.alerts.end}}
+
 Enable logical replication by setting `wal_level` to `logical` in `postgresql.conf` or in the SQL shell. For example:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
 ALTER SYSTEM SET wal_level = 'logical';
 ~~~
+
+Create a publication for the tables you want to replicate. Do this **before** creating the replication slot. 
+
+To create a publication for all tables:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE PUBLICATION molt_publication FOR ALL TABLES;
+~~~
+
+To create a publication for specific tables:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE PUBLICATION molt_publication FOR TABLE employees, payments, orders;
+~~~
+
+Create a logical replication slot:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT pg_create_logical_replication_slot('molt_slot', 'pgoutput');
+~~~
+
+##### Verify logical replication setup
+
+Verify the publication was created successfully:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM pg_publication;
+~~~
+
+~~~
+  oid  |     pubname      | pubowner | puballtables | pubinsert | pubupdate | pubdelete | pubtruncate | pubviaroot
+-------+------------------+----------+--------------+-----------+-----------+-----------+-------------+------------
+ 59084 | molt_publication |       10 | t            | t         | t         | t         | t           | f
+~~~
+
+Verify the replication slot was created:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT * FROM pg_replication_slots;
+~~~
+
+~~~
+ slot_name |  plugin  | slot_type | datoid | database | temporary | active | active_pid | xmin | catalog_xmin | restart_lsn | confirmed_flush_lsn | wal_status | safe_wal_size | two_phase
+-----------+----------+-----------+--------+----------+-----------+--------+------------+------+--------------+-------------+---------------------+------------+---------------+-----------
+ molt_slot | pgoutput | logical   |  16385 | molt     | f         | f      |            |      |         2261 | 0/49913A20  | 0/49913A58          | reserved   |               | f
+~~~
 </section>
 
 <section class="filter-content" markdown="1" data-scope="mysql">
-For MySQL **8.0 and later** sources, enable [global transaction identifiers (GTID)](https://dev.mysql.com/doc/refman/8.0/en/replication-options-gtids.html) consistency. Set the following values in `mysql.cnf`, in the SQL shell, or as flags in the `mysql` start command:
+Enable [global transaction identifiers (GTID)](https://dev.mysql.com/doc/refman/8.0/en/replication-options-gtids.html) and configure binary logging. Set `binlog-row-metadata` or `binlog-row-image` to `full` to provide complete metadata for replication.
```

Comment:
I think it may be worth calling out that it's also important to tune binlog retention: https://dba.stackexchange.com/a/206602

This can impact if the data from the GTID you specify is still available or if it's now purged/rotated. It's important to note that if using something like AWS RDS or GCP CloudSQL, there are provider specific ways they handle this:
- https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/mysql-stored-proc-configuring.html
- https://discuss.google.dev/t/cloud-sql-mysql-database-instance-binlog-is-keep-increasing/124841/2 (binlog_expire_logs_seconds flag)